demo/src/main/java/com/orange/ods/demo/ui/components/buttons/ButtonsContained.kt

```diff
@@ -26,15 +26,10 @@ import com.orange.ods.demo.ui.utilities.composable.Title
 import com.orange.ods.utilities.extension.fullName
 
 @Composable
-fun ButtonsHighEmphasis() {
-    Title(R.string.component_buttons_high_emphasis_primary, withHorizontalPadding = true)
-    TechnicalText(text = OdsButtonStyle.Primary.fullName, withHorizontalPadding = true)
-    ContainedButtons(style = OdsButtonStyle.Primary)
+fun ButtonsPrimary() = ContainedButtons(style = OdsButtonStyle.Primary)
 
-    Title(R.string.component_buttons_high_emphasis_standard, withHorizontalPadding = true)
```

Comment:
`component_buttons_high_emphasis_primary` and `component_buttons_high_emphasis_standard` are now unused.